simple-generated/Cargo.toml

```diff
@@ -6,4 +6,6 @@ license = "MIT"
 edition = "2021"
 
 [dependencies]
+crossterm = "0.28.1"
 ratatui = "0.28.1"
+color-eyre = "0.6.3"
```

Comment:
I personally don't like this—I believe we should leave the decision to use an error-handling library up to the user.

Reply:
The only reason I don't like color-eyre is because it's autocomplete messes with ratatui's styles. If it weren't for that, I'd always use color-eyre. I still think it is a good default for the template.